sdk/monitor/query/azlogs/client.go

```diff
@@ -76,20 +81,23 @@ func (client *Client) queryBatchHandleResponse(resp *http.Response) (QueryBatchR
 	return result, nil
 }
 
-// QueryResource - Executes an Analytics query for data in the context of a resource. Here [https://learn.microsoft.com/azure/azure-monitor/logs/api/azure-resource-queries]
-// is an example for using POST with an Analytics
-// query.
-// If the operation fails it returns an *azcore.ResponseError type.
+// QueryResource - Execute an Analytics query using resource ID
 //
-// Generated from API version 2022-10-27
+// Executes an Analytics query for data in the context of a resource.
+// [Here](https://learn.microsoft.com/azure/azure-monitor/logs/api/azure-resource-queries)
+// is an example for using POST with an Analytics query.
+// If the operation fails it returns an *azcore.ResponseError type.
 //   - resourceID - The identifier of the resource.
-//   - body - The Analytics query. Learn more about the Analytics query syntax [https://azure.microsoft.com/documentation/articles/app-insights-analytics-reference/]
-//   - options - QueryResourceOptions contains the optional parameters for the Client.QueryResource method.
-func (client *Client) QueryResource(ctx context.Context, resourceID string, body QueryBody, options *QueryResourceOptions) (QueryResourceResponse, error) {
+//   - body - The Analytics query. Learn more about the
+//     [Analytics query syntax](https://azure.microsoft.com/documentation/articles/app-insights-analytics-reference/)
+//   - opts - QueryResourceOptions contains the optional parameters for the Client.QueryResource method.
+func (client *Client) QueryResource(ctx context.Context, resourceID string, body QueryBody, opts *QueryResourceOptions) (QueryResourceResponse, error) {
```

Comment:
This renaming of `options` to `opts` is an emitter bug. While harmless, it does introduce a diff in the APIView which is unfortunate.

https://github.com/Azure/autorest.go/pull/1649